blogposts/2024-07-18-running-rnw-from-a-mac.md

```diff
@@ -0,0 +1,86 @@
+
+I thoroughly enjoyed my experience last year as a speaker. I also loved the conversations I had with developers at the conference after my talk. It was so great to hear how many of you were excited by React Native for Desktop and that our message that Microsoft has bet on this platform has landed.
+
+In all the conversations, one comment rang especially clearly: _*"React Native for Windows sounds great! I'd love to give it a try on my product…but I use a Mac as my development device."*_ Many of you were excited by React Native for Windows, but thought you were unable to give the platform a try because you use a Mac as your development machine and didn't have access to a Windows device.
+
+The React Native for Windows team has heard the community feedback. While running on a Windows device will be the best experience for developing on Windows; we recognize that not everyone involved in a development team can have access to multiple devices to develop for different operating systems. We also see that Mac computers are used widely within the React Native developer community.
+
+In today's blog post, I'm going to walk through a few options for how to develop React Native for Windows applications from your Mac computer because owning a Windows device is not a requirement to build and ship Windows experiences.
+
+To build Windows experiences from a non-Windows device, we'll leverage virtual machine software. There are several options you have as a developer to spin-up a Windows VM from a Mac computer including Parallels, VirtualBox, VMWare, and Microsoft Dev Box to name a few. In this post, I'll walk through my experience using Parallels and Microsoft Dev Box.
+
+## Creating a Windows VM using Parallels
+
+The first Windows VM option I tried was [Parallels](https://www.parallels.com/). Parallels is a software which can be locally installed on a developer's MacOS machine to be used to create virtual machines of a variety of operating systems.
+
+After signing up for the [Parallels free trial](https://www.parallels.com/products/desktop/trial/), I was given a download of the Parallels Desktop software and installation instructions. After Parallels Desktop was installed and running, the software prompted me to install Windows 11. From there, I was able to run a Windows VM.
+
+With my virtual machine running, I was able to use the VM as if I was on a Windows computer. From here, I could follow the ["System Requirements"](https://microsoft.github.io/react-native-windows/docs/rnw-dependencies) and ["Getting Started"](https://microsoft.github.io/react-native-windows/docs/getting-started) documentation for React Native for Windows. Parallels had access to my local files, so I did not need to redownload any source code to get started developing. When I was finished with my development session, I was able to close the virtual machine and the machine's state was saved for next time.
+
+![](assets/2024-07-18-running-rnw-from-a-mac/Screenshot-Parallels.jpg)
```

Comment:
We had some trouble with assets last time, I think due to file size. I think you should be ok though.